### DISCUSSION TITLE
Blue tint screen after fresh install

### DISCUSSION THREAD

**Discussion by AlejoGarcia** [elementary/triage#207] [Q&A]
I have a problem with my screen: in the log in phase it becomes blue (only B of RGB is displayed). I have seen other people with this problem and have one patch solution but I don't know the cause nor a permanent fix.

**Comment by migtorr** [comment]
workaround https://reddit.leptons.xyz/r/elementaryos/comments/11osvpy/blue_tint_on_fresh_install/jbx69ax/?context=3
